index.js

```diff
@@ -2,12 +2,23 @@
 const {ContextualIdentityService} = require('resource://gre/modules/ContextualIdentityService.jsm');
 const { Cc, Ci, Cu, Cr } = require('chrome');
 
+Cu.import("resource://gre/modules/Services.jsm");
+
 const tabs = require('sdk/tabs');
 const webExtension = require('sdk/webextension');
 const { viewFor } = require("sdk/view/core");
 var windowUtils = require('sdk/window/utils');
 var tabsUtils = require('sdk/tabs/utils');
 
+const { browserWindows: windows } = require("sdk/windows");
```

Comment:
you make my life easier if you do : `var windows = require("sdk/windows");`
ad then you do:

for (let window in windows.browserWindows) {
  ...